src/pages/trade/ui/positions/index.tsx

```diff
@@ -1,8 +1,11 @@
 'use client';
 
 import Link from 'next/link';
-import { useEffect } from 'react';
+import orderBy from 'lodash/orderBy';
+import { useEffect, useState, useCallback } from 'react';
 import { connectionStore } from '@/shared/model/connection';
+import { ChevronDown } from '@styled-icons/evaicons-solid/ChevronDown';
+import { ChevronUp } from '@styled-icons/evaicons-solid/ChevronUp';
```

Comment:
question: why using new library instead of lucide-icons?

suggestion: keep using lucide-icons, maybe consult with Artur about the best design decision

Reply:
Updated them.